components/brave_vpn/browser/connection/wireguard/win/brave_vpn_wireguard_service/status_tray/resources/BUILD.gn

```diff
@@ -3,9 +3,35 @@
 # License, v. 2.0. If a copy of the MPL was not distributed with this file,
 # You can obtain one at https://mozilla.org/MPL/2.0/.
 
+import("//base/win/embedded_i18n/generate_embedded_i18n.gni")
+
 source_set("resources") {
   inputs = [
     "interactive_resource.h",
     "resources_en.rc",
   ]
+  deps = [ ":strings" ]
+}
+
+generate_embedded_i18n("generate_strings") {
+  visibility = [ ":strings" ]
+  extractor_datafile = "//brave/components/brave_vpn/browser/connection/wireguard/win/brave_vpn_wireguard_service/status_tray/resources/create_wireguard_string_rc.py"
+  grdfile_folder = "//chrome/app/"
```

Comment:
it seems layer violation?

Reply:
This is folder where it will take grd file to extracat strings and create a header in gen. The header created inside current layer  so should be ok. Moving this code to `chrome` will produce the header inside chrome and we will not be able to use it inside the component. cc @brave/string-reviewers-team

Reply:
Got it.
Another question - In this PR, new strings are added into `brave/app/brave_strings.grd`.
Curious they could be added into grd files in `brave/component/` instead?

Reply:
when I try to do this there's next error:
```
[`brave_components_resources_af.xtb`](ninja: error: '../../brave/components/resources/strings/brave_components_resources_af.xtb', needed by 'gen/brave/components/brave_vpn/browser/connection/wireguard/win/brave_vpn_wireguard_service/status_tray/resources/wireguard_strings.h', missing and no known rule to make it)
```

if I created that file manually then got same error for `brave_components_resources_as.xtb`. Looks like we do not support all languages, maybe @mkarolin can advise something how to fix it or keep current way.

Reply:
We can't add new strings directly into `brave/app/brave_strings.grd`. This file is auto updated from `src/chrome/app/chromium_strings.grd` (+2 entries we add in script), so your changes would be removed when we run rebase the next time.

`brave_component_resources` seems like a good option, but as you noticed we don't support all the languages that Chromium supports (see https://github.com/brave/brave-browser/wiki/Strings-and-Localization#information-for-submitting-localization-orders section - the languages we support are bolded).

Reply:
Thanks @mkarolin , updated code to exclude unused locales from component grds, but there are some errors:

If I use `brave_component_resources` throws:
```
There were files that exist on disk but were not found in the .grd file '../../brave/components/resources/brave_components_resources.grd':
../../brave/components/resources/strings/brave_components_resources_am.xtb
../../brave/components/resources/strings/brave_components_resources_ar.xtb
../../brave/components/resources/strings/brave_components_resources_bg.xtb
....
../../brave/components/resources/strings/brave_components_resources_zh-TW.xtb
[126/31408] CXX obj/mojo/core/ports/ports/node.obj
```

If I use `brave_component_strings` throws: 
```
ninja: error: '../../brave/components/resources/strings/brave_components_strings_am.xtb', needed by 'gen/brave/components/brave_vpn/browser/connection/wireguard/win/brave_vpn_wireguard_service/status_tray/resources/wireguard_strings.h', missing and no known rule to make it
ninja: Entering directory `D:\repos\brave\brave-browser\src\out\Component'
```

Reply:
Finally it required 2 things to make it worked:
1) Rename `components\resources\strings\brave_components_resources_XX.xtb` - > `components\resources\strings\brave_components_strings_XX.xtb` 
2) Move Wireguard service strings out of `brave_vpn_strings.grdp` to `components\resources\brave_components_strings.grd` because `create_string_rc.py` doesnt work with grdp.

It looks working now, please take a look @mkarolin

Reply:
This looks good to me @spylogsster. Could you, please, also update the comment here https://github.com/brave/brave-core/blob/master/build/commands/lib/l10nUtil.js#L57.

Reply:
вщту